diff --git a/node_modules/@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol b/node_modules/@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol
index fda8dd1..c2775a9 100644
--- a/node_modules/@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol
+++ b/node_modules/@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol
@@ -19,8 +19,8 @@ import "../TimelockController.sol";
  * _Available since v4.3._
  */
 abstract contract GovernorTimelockControl is IGovernorTimelock, Governor {
-    TimelockController private _timelock;
-    mapping(uint256 => bytes32) private _timelockIds;
+    TimelockController public _timelock;
+    mapping(uint256 => bytes32) public _timelockIds;
 
     /**
      * @dev Emitted when the timelock controller used for proposal execution is modified.
